### DISCUSSION TITLE
Support for beancount files

### DISCUSSION THREAD

**Discussion by bhi5hmaraj** [ananthakumaran/paisa#187] [General]
I've been using beancount and fava for a few years now and found it easier to implement custom importers in python. As of now paisa only supports ledger as a source. I could use something like https://github.com/beancount/beancount2ledger , but I think it might cause subtle bugs which would be harder to debug. 

How hard would it be to support beancount as a source for paisa?

**Comment by ananthakumaran** [comment]
https://paisa.fyi/reference/ledger-cli/

**Comment by bhi5hmaraj** [comment]
Thanks!
